lib/test_ids.rb

```diff
@@ -15,156 +15,160 @@ module TestIds
-
-      return if @configuration[@configuration_id]
-
-      @configuration[@configuration_id] = Configuration.new(@configuration_id)
-
-      config = @configuration[@configuration_id]
-
-      yield config
-
-      config.validate!
-
-      initialize_git
-    end
-
-    ## Can be called in place of TestIDs.configure to change the configuration from
-    ## the one that was originally supplied.
-    ## It is expected that this is mainly useful for testing purposes only.
-    # def reconfigure(id = nil, options = {}, &block)
-    #  id, options = nil, id if id.is_a?(Hash)
-
-    #  @configuration_id = id || options[:id] || :not_specified
-
-    #  @configuration ||= {}
-
-    #  old = @configuration[@configuration_id]
-    #  new = Configuration.new(@configuration_id)
-    #  new.instance_variable_set('@allocator', old.allocator)
-    #  new.allocator.instance_variable_set('@config', new)
-    #  @configuration[@configuration_id] =  new
-
-    #  yield new
-
-    #  new.validate!
-    # end
-
-    def configured?
-      !!@configuration_id
-    end
-
-    def initialize_git
-      @git_initialized ||= begin
-        if repo
-          @git = Git.new(local: git_database_dir, remote: repo)
-          git.get_lock if publish?
-        end
-        true
-      end
-    end
-
-    # Returns a full path to the database file for the given id, returns nil if
-    # git storage has not been enabled
-    def database_file(id)
-      if repo
-        if id == :not_specified || !id || id == ''
-          f = 'store.json'
-        else
-          f = "store_#{id.to_s.downcase}.json"
-        end
-        "#{git_database_dir}/#{f}"
-      end
-    end
-
-    def git_database_dir
-      @git_database_dir ||= begin
-        d = "#{Origen.app.imports_directory}/test_ids/#{Pathname.new(repo).basename}"
-        FileUtils.mkdir_p(d)
-        d
-      end
-    end
-
-    def git
-      @git
-    end
-
-    def repo=(val)
-      return if @repo && @repo == val
-      if @repo && @repo != val
-        fail 'You can only use a single test ids repository per program generation run, one per application is recommended'
-      end
-      if @configuration
-        fail 'TestIds.repo must be set before creating the first configuration'
-      end
-      @repo = val
-    end
-
-    def repo
-      @repo
-    end
-
-    def publish?
-      @publish ? @publish == :save : true
-    end
-
-    def publish=(val)
-      return if @publish && publish? == val
-      if @publish && publish? != val
-        fail 'You can only use a single setting for publish per program generation run'
-      end
-      if @configuration
-        fail 'TestIds.publish must be set before creating the first configuration'
-      end
-      unless [true, false].include?(val)
-        fail 'TestIds.publish must be set to either true or false'
-      end
-      @publish = val ? :save : :dont_save
-    end
+   # Allocates a number to the given test and returns a new hash containing
```

Comment:
What's caused this diff, editing in Windows or something? Can you get rid of it?